### PR TITLE
[DOCS-7114] Update remote partitioning commands for Search Enterprise

### PR DESCRIPTION
**Pull request by Prosune**

### CODE REVIEW

File: search-enterprise/3.2/admin/index.md [hunk 1]
```diff
@@ -313,19 +313,24 @@ Both strategies split the specified range into multiple ranges depending on the
 _Manager_:
 
 ```shell
-java -jar alfresco-elasticsearch-reindexing-3.2.0-app.jar \
+java -jar alfresco-elasticsearch-reindexing-4.0.0-app.jar \
```
Comment (tpage-alfresco) [inline search-enterprise/3.2/admin/index.md:316]: The latest released version for 3.2 is now 3.2.1.
```suggestion
java -jar alfresco-elasticsearch-reindexing-3.2.1-app.jar \
```

File: search-enterprise/3.3/admin/index.md [hunk 2]
```diff
@@ -374,19 +374,24 @@ Both strategies split the specified range into multiple ranges depending on the
 _Manager_:
 
 ```shell
-java -jar alfresco-elasticsearch-reindexing-3.3.0-app.jar \
+java -jar alfresco-elasticsearch-reindexing-4.0.0-app.jar \
```
Comment (tpage-alfresco) [inline search-enterprise/3.3/admin/index.md:377]:
```suggestion
java -jar alfresco-elasticsearch-reindexing-3.3.2-app.jar \
```

File: search-enterprise/latest/admin/index.md [hunk 3]
```diff
@@ -381,12 +381,17 @@ java -jar alfresco-elasticsearch-reindexing-4.0.0-app.jar \
   --alfresco.reindex.batchSize=100  \
```
Comment (tpage-alfresco) [inline search-enterprise/latest/admin/index.md:381]: I'm not sure if you want to try to keep the version updated or not (it's probably an impossible job), but the latest 4.0.x version is 4.0.1.

Comment (Prosune) [reply search-enterprise/latest/admin/index.md:381]: Let's keep it as current as we can.

**Review by tpage-alfresco — APPROVED** [review]

**Pull request merged**